### ISSUE TITLE
Unsatisfiable Constraint when trying to make plot size responsive within panels

### ISSUE THREAD

**Issue by hhprogram** [bokeh/bokeh#7260]
Bokeh version: 0.12.10

I am trying to have a tab layout with a figure within each panel. I want each figure to have responsive plot sizing (ie be able to execute `sizing_mode='stretch_both'` etc..). However, when I put in responsive arguments for the parameter `sizing_mode` I get a Bokeh Error: Unsatisfiable constraint. I've tried applying the same `sizing_mode` to the Panel and Tabs object as well but still get the same error.

Note: I am able to manually make the plot as large as the window screen, via `plot_width`, however that is not desirable as that is not dynamic. 

```
from bokeh.models.widgets import Panel, Tabs
from bokeh.io import output_file, show
from bokeh.plotting import figure

output_file("slider.html")

p1 = figure(sizing_mode='stretch_both')
p1.circle([1, 2, 3, 4, 5], [6, 7, 2, 4, 5], size=20, color="navy", alpha=0.5)
tab1 = Panel(child=p1, title="circle")

p2 = figure(sizing_mode='stretch_both')
p2.line([1, 2, 3, 4, 5], [6, 7, 2, 4, 5], line_width=3, color="navy", alpha=0.5)
tab2 = Panel(child=p2, title="line")

tabs = Tabs(tabs=[ tab1, tab2 ])

show(tabs)
```

**Comment by mattpap** [comment]
Which version of bokeh?

**Comment by hhprogram** [comment]
Sorry - see updated post

**Comment by bryevdv** [comment]
@hhprogram can you try with the latest? I have no expectation whether or not it will be fixed there but that is where we will have to start from.

**Comment by mattpap** [comment]
Nothing changed with respect to sizing modes, so there should be no difference.

**Comment by hhprogram** [comment]
Sorry for the delay. But @mattpap  is correct in that the problem is not fixed. However, I no longer see the 'Unsatisfiable Constraint' error text pop up in my html. Now it's just results in a white blank web page.

**Comment by bryevdv** [comment]
This might be a dupe, but I'm not certain.

**Comment by zpz** [comment]
I have similar issues. I'm embedding bokeh 'server_document' on a web page and I want the 'sizing_model="scale_width" behavior. I have a 'column' containing a Slider and a plot. This column alone works well. If I put it in a Panel, it does not appear at all (this I don't understand---can't Panel be used by itself?). If I put the Panel in a Tabs, the 'column' in the Panel shrinks to a tiny bit of something. The 'title' bar of the Panel also shrinks but not that bad---at least it's readable.

**Comment by hhprogram** [comment]
Any update or any aid I can provide for this issue?

**Comment by darahsten** [comment]
Yes, sizing mode scale_both is broken. Tried and failed too. had to revert to fixed

**Comment by brcnblc** [comment]
I experience the same issue, after resizing browser window to tablet or phone size. I use scale_width, and column layout containing 3 childs (external toolbar, graph1, graph2) 

> Uncaught Error: unsatisfiable constraint: Canvas(97CE6AF7B47D49C3AA6E3CC142472B71).height - 90.6 == 0

It also gives following error after :

Error: dual optimize failed

**Comment by bryevdv** [comment]
The original code above is fixed by #8085
